docs/io/fits/usage/unfamiliar.rst

```diff
@@ -136,7 +137,7 @@ type specification (i.e. the value of the TFORM keyword) uses an extra letter
 
     rPt(max)
 
-where r is 0, 1, or absent, t is one of the letter code for regular table data
+where r is the column length (it can be absent), t is one of the letter code for regular table data
```

Comment:
I wouldn't describe this as "the column length".  That implies the number of rows, typically.  
Instead I would just say "r may be 0 or 1, but is typically absent, as it is not applicable to variable length arrays".

Please also try to wrap the text so that the lines are under 80 characters.  Thanks!